spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/data/jpa/JpaRepositoriesAutoConfiguration.java

```diff
@@ -66,10 +69,23 @@
 @ConditionalOnMissingBean({ JpaRepositoryFactoryBean.class, JpaRepositoryConfigExtension.class })
 @ConditionalOnProperty(prefix = "spring.data.jpa.repositories", name = "enabled", havingValue = "true",
 		matchIfMissing = true)
-@Import(JpaRepositoriesRegistrar.class)
 @AutoConfigureAfter({ HibernateJpaAutoConfiguration.class, TaskExecutionAutoConfiguration.class })
 public class JpaRepositoriesAutoConfiguration {
 
+	@Configuration(proxyBeanMethods = false)
+	@ConditionalOnClass(EnversRevisionRepositoryFactoryBean.class)
```

Comment:
Is this a good target for the conditional? Mostly a style question.

I would have favored `org.springframework.data.envers.repository.support.EnversRevisionRepository` but it is deprecated. Another option could be `org.springframework.data.envers.repository.support.EnversRevisionRepositoryImpl`.